vspace/src/main/webapp/resources/extra/Home.css

```diff
@@ -549,13 +564,31 @@
     margin: 5px;
 }
 
+.aboutDescription {
+    padding-top: 26px;
+    padding-left: 5px;
+    padding-right: 5px;
+    padding-bottom: 5px;
+    word-break: break-word;
+    text-align: justify;
+    margin: 5px;
+}
+
 #rightContent .closebtn {
     position: absolute;
     top: 8px;
     right: 3px;
     margin-left: 50px;
 }
 
+#aboutContent .closebtn {
+    position: absolute;
+    top: 8px;
+    right: 3px;
+    margin-left: 50px;
+}
+
+
```

Comment:
are these css changes still needed?

Reply:
No, removed

Reply:
pelase remove them completely so the file doesn't show up as changed anymore (see the checklist you checked of)